.github/workflows/lint.yml

```diff
@@ -0,0 +1,41 @@
+name: Lint
+
+on:
+  push:
+
+env:
+  TERRAFORM_VERSION: "1.6.1"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  linting:
+    name: Linter
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Install Terraform
+        uses: hashicorp/setup-terraform@v2.0.3
+        with:
+          terraform_version: ${{ env.TERRAFORM_VERSION }}
+
+      - name: Run Terraform format
+        run: terraform fmt -recursive -check
+
+      - name: Run trivy linter
+        uses: aquasecurity/trivy-action@0.12.0
+        with:
+          image-ref: '.'
```

Comment:
Do you have any examples where the linter fails, ie trivy? 🙏

Reply:
This is an example 
![image](https://github.com/nvminhtue/tyson_devops_ic/assets/42540264/63e78db6-7892-4e08-92d7-dc8a54f23886)
Updated to description